src/test/java/emissary/util/ShortNameComparatorTest.java

```diff
@@ -37,13 +37,23 @@ public void testNonsenseNames() {
         assertEquals("Bad components sort in alpha order", "foo-att-abc", l.get(0).shortName());
     }
 
+    @Test
+    public void testNoChildren() {
```

Comment:
Since `ShortNameComparator` is only used to sort items from a single family tree, this test is probably unnecessary, since a family tree can have a single one item that is not a child.

If we want to assert specific behavior, the message in case of failed assertion might be more accurate as "IBDOs that are not children sort in insertion order"